lib/FieldType/Mapper/FloatFormMapper.php

```diff
@@ -61,7 +71,54 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
     }
 
     /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $validatorConfiguration = $fieldDefinition->getValidatorConfiguration();
+        $formConfig = $fieldForm->getConfig();
+        $attributes = [
+            'step' => 1,
+        ];
+
+        if (null !== $validatorConfiguration['FloatValueValidator']['minFloatValue']) {
```

Comment:
I'd move that to a private method, like `mapValidatorConfiguration($validatorConfiguration)`.